planner/pom.xml

```diff
@@ -14,10 +14,11 @@
 		<project.reporting.outputEncoding>${sourceEncoding}</project.reporting.outputEncoding>
 
 		<java.version>1.8</java.version>
+		<log4j.version>2.16.0</log4j.version>
```

Comment:
Just making sure this isn't a typo and that we intend to bump to `2.16.0`?

Reply:
It is not a typo. The guidance had changed after making the PR and its title to use 2.16.0 instead of 2.15.0 I will just update the PR title as well to prevent confusion.